docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3'
+services:
+  db:
+    image: postgres
+    restart: always
+    networks:
+      - potatonotes
+    environment:
+      POSTGRES_USER: notes
+      POSTGRES_DB: potatonotes
+      POSTGRES_PASSWORD: notes
+    volumes:
+    - db-data:/var/lib/postgresql/data
+
+  api:
+    image: atechnohazard/potatosync:latest
+    restart: always
+    depends_on:
+      - db
+    networks:
+      - potatonotes
+    volumes:
+      - ./.env:/go/bin/.env
```

Comment:
https://docs.docker.com/compose/environment-variables/#the-env-file is the recommended way :)

Reply:
Because the variables are read from an .env instead of from the environment, this approach doesnt work unless i add the HEROKU variable to the .env. Otherwise i will get this:
```
bas@pop-os:~/Documents/Projects/potato2$ sudo docker-compose up --force-recreate
Recreating potato2_db_1 ... done
Recreating potato2_api_1 ... done
Attaching to potato2_db_1, potato2_api_1
db_1   | 
db_1   | PostgreSQL Database directory appears to contain a database; Skipping initialization
db_1   | 
db_1   | 2020-03-17 18:47:47.582 UTC [1] LOG:  starting PostgreSQL 12.2 (Debian 12.2-2.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
db_1   | 2020-03-17 18:47:47.582 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db_1   | 2020-03-17 18:47:47.582 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db_1   | 2020-03-17 18:47:47.585 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db_1   | 2020-03-17 18:47:47.607 UTC [26] LOG:  database system was shut down at 2020-03-17 18:46:10 UTC
db_1   | 2020-03-17 18:47:47.624 UTC [1] LOG:  database system is ready to accept connections
api_1  | 2020/03/17 18:47:48 open .env: no such file or directory
api_1  | 2020/03/17 18:47:49 open .env: no such file or directory
potato2_api_1 exited with code 1
potato2_api_1 exited with code 1
```

Reply:
You're free to modify the code to achieve desired functionality, I will merge it as long as it works.